StreamChatUITestsAppUITests/Robots/ParticipantRobot.swift

```diff
@@ -195,13 +172,61 @@ final class ParticipantRobot: Robot {
         return self
     }
     
+    @discardableResult
+    func sendGiphy() -> Self {
+        waitForChannelQueryUpdate()
+        server.websocketMessage(
+            channelId: server.currentChannelId,
+            messageId: TestData.uniqueId,
+            messageType: .ephemeral,
+            eventType: .messageNew,
+            user: participant()
+        )
+        return self
+    }
+    
+    @discardableResult
+    func replyWithGiphy() -> Self {
+        let quotedMessage = server.lastMessage
+        let quotedMessageId = quotedMessage?[MessagePayloadsCodingKeys.id.rawValue] as? String
+        server.websocketMessage(
+            channelId: server.currentChannelId,
+            messageId: TestData.uniqueId,
+            messageType: .ephemeral,
+            eventType: .messageNew,
+            user: participant()
+        ) { message in
+            message?[MessagePayloadsCodingKeys.quotedMessageId.rawValue] = quotedMessageId
+            message?[MessagePayloadsCodingKeys.quotedMessage.rawValue] = quotedMessage
+            return message
+        }
+        return self
+    }
+    
+    @discardableResult
+    func replyWithGiphyInThread(alsoSendInChannel: Bool = false) -> Self {
+        let parentId = threadParentId ?? (server.lastMessage?[MessagePayloadsCodingKeys.id.rawValue] as? String)
+        server.websocketMessage(
+            channelId: server.currentChannelId,
+            messageId: TestData.uniqueId,
+            messageType: .ephemeral,
+            eventType: .messageNew,
+            user: participant()
+        ) { message in
+            message?[MessagePayloadsCodingKeys.parentId.rawValue] = parentId
+            message?[MessagePayloadsCodingKeys.showReplyInChannel.rawValue] = alsoSendInChannel
```

Comment:
Why these values are not among `server.websocketMessage` input params?

Reply:
We try to keep `websocketMessage` method as common and "tiny" as possible because not all messages have params such as `parentId`, etc. We handle it through `intercept` param, which is used in multiple places across the mock server.

Reply:
Ok, understood 👍At some point when we bring more WS events to the mock server, we might need to switch to the approach and have many methods, one per each event type (cc @bielikb)